samples/test/translate_model_management_test.js

```diff
@@ -0,0 +1,106 @@
+const {AutoMlClient} = require('@google-cloud/automl');
+
+const cp = require('child_process');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+// const DATASET_ID = 'TRL8522556519449886720';
+const LIST_MODEL_REGION_TAG = 'list_models';
+const GET_MODEL_REGION_TAG = 'get_model';
+const LIST_MODEL_EVALUATION_REGION_TAG = 'list_model_evaluations';
+const GET_MODEL_EVALUATION_REGION_TAG = 'get_model_evaluation';
+const LIST_OPERATION_STATUS_REGION_TAG = 'list_operation_status';
+const GET_OPERATION_STATUS_REGION_TAG = 'get_operation_status';
+
+// const CREATE_MODEL_REGION_TAG = 'translate_create_model';
+// const DELETE_MODEL_REGION_TAG = 'delete_model';
+
+describe('Automl Translate Model Tests', () => {
+  const client = new AutoMlClient();
+
+  it('should list and get model, list and get model evaluations', async () => {
+    const projectId = await client.getProjectId();
+
+    // list models
+    const list_model_output = execSync(
+      `node ${LIST_MODEL_REGION_TAG}.js ${projectId}`
+    );
+    assert.match(list_model_output, /Model id:/);
+    const modelId = list_model_output.split('Model id: ')[1].split('\n')[0];
+
+    // get model
+    const get_model_output = execSync(
+      `node ${GET_MODEL_REGION_TAG}.js ${projectId} ${modelId}`
+    );
+    assert.match(get_model_output, /Model id/);
+
+    // list model evaluations
+    const list_model_eval_output = execSync(
+      `node ${LIST_MODEL_EVALUATION_REGION_TAG}.js ${projectId} ${modelId}`
+    );
+    assert.match(list_model_eval_output, /Model evaluation name/);
+    const modelEvaluationId = list_model_eval_output
+      .split(`${modelId}/modelEvaluations/`)[1]
+      .split('\n')[0];
+
+    // get model evaluation
+    const get_model_eval_output = execSync(
+      `node ${GET_MODEL_EVALUATION_REGION_TAG}.js ${projectId} ${modelId} ${modelEvaluationId}`
+    );
+    assert.match(get_model_eval_output, /Model evaluation name/);
+  });
+
+  it('should list / get operation status', async () => {
+    const projectId = await client.getProjectId();
+
+    const list_output = execSync(
+      `node ${LIST_OPERATION_STATUS_REGION_TAG}.js ${projectId}`
+    );
+    assert.match(list_output, /Operation details/);
+    const operation_id = list_output.split('Name: ')[1].split('\n')[0];
+
+    const get_output = execSync(
+      `node ${GET_OPERATION_STATUS_REGION_TAG}.js ${operation_id}`
+    );
+    assert.match(get_output, /Operation details/);
+  });
+
+  // it('should create a model', async () => {
+  //   const projectId = await client.getProjectId();
+  //   const create_output = execSync(`node ${CREATE_MODEL_REGION_TAG}.js ${projectId} ${DATASET_ID} translation_test_create_model`)
+
+  //   assert.match(create_output, /Training started/);
+
+  //   const operationId = create_output.split('Training operation name: ')[1].split('\n')[0]
+  //   client.operationsClient.cancelOperation(operationId);
+  // });
+
+  // it('should delete a model', async () => {
```

Comment:
doing the delete on the not found is good - we could also mock the response of the deletion

Reply:
How do I try like a try catch. 
At first the error was captured via output, but because the model doesn't exist (fake model id) I want to catch the error that the model wasn't found but the rest of the request was valid.